examples/window/low_power.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    prelude::*,
+    window::RequestRedraw,
+    winit::{ControlFlow, WinitConfig},
+};
+
+/// This example illustrates how to run in low power mode, useful for making desktop applications.
```

Comment:
How does this strategy interact with async tasks, like e.g. slicing a 3D print, that you might want to perform while the app is minimized?

Reply:
I assume that if work is spawned to a task pool, this won't prevent progress assuming the task pool is also being polled from the other thread. If you really need to keep running the world, you can just send a redraw request event.

Reply:
I agree with you, but we should test and demonstrate this in the example :)

Reply:
Hmm, how do you think we could test? Spawn a task set on a timer for 5 sec-> minimize for 5 secs -> restore window and see if the task completes immediately.

Reply:
> Hmm, how do you think we could test? Spawn a task set on a timer for 5 sec-> minimize for 5 secs -> restore window and see if the task completes immediately.

Yep that sounds perfect!